src/languages/en.js

```diff
@@ -935,16 +935,16 @@ export default {
     },
     getAssistancePage: {
         title: 'Get assistance',
-        subtitle: 'Have questions, or need help?',
-        description: 'We have answers! Choose from the support options below:',
+        subtitle: 'We\'re here to clear your path to greatness!',
+        description: 'Choose from the support options below:',
         chatWithConcierge: 'Chat with Concierge',
         requestSetupCall: 'Request a setup call',
         questionMarkButtonTooltip: 'Get assistance from our team',
     },
     requestCallPage: {
         title: 'Request a call',
-        subtitle: 'Need help?',
-        description: 'Our team is ready to help each step of the way. Enter your name and phone number, and we\'ll give you a call back asap.',
+        subtitle: 'Need help, or a demo?',
+        description: 'Our team is ready to help each step of the way. Enter your name and phone number, and we\'ll give you a call back ASAP.*',
```

Comment:
Is this `*` intentional?

Reply:
Yes: https://github.com/Expensify/App/pull/11397#issuecomment-1261740093

Reply:
Do we want it for the spanish version too then?

Reply:
cc @kadiealexander

Reply:
Hmmm... yeah I think we would right @kadiealexander?